### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by lucasnordic**
Updated ReadMe so that it follows our new project goals. #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 # Group 10
 
 ## What you are going to make?
-- GeoBot 	
-- This program will let users control a smart vehicle and possibly turn on automated exploring.  
-- It will be able to create and map an unidentified area.
+- Tank - Total Annihilation Not Komplete
```
Comment (hannahshiels) [inline README.md:4]: Should it not be spelled 'Complete' 😆

File: README.md [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+- It will give users of the Android app the possibility of detecting objects by looking at a camera feed. This camera feed is recieved from the bot they control in SMCE
+- This program will let users control a smart bot in SMCE with the objective of crashing into other SMCE bots.
+- It will possibly be able to do automated bot hunting.  
 
 ## Why will you make it? What problem does it solve?
-- GeoBot will automatically explore and create a map of an unknown area 
-- It can be used to explore places that are potentially dangerous for humans
+- Tank will solve the issue of how to detect objects, and will assist App users in hunting down these objects and crash into them(mainly other bots/smartcars)
+- It can be used to cause mayham and destruction
```
Comment (hannahshiels) [inline README.md]: I think mayhem is spelled with an e 🤔

File: README.md (hunk 1 above)
Comment (lucasnordic) [reply README.md:4]: Eyy, I didn't have the heart to change Tank into Tanc you know :P. See the word as a sweedishified version of complete

File: README.md (hunk 2 above)
Comment (lucasnordic) [reply README.md]: Hmm, indeed 🤔

**Review by hannahshiels — APPROVED** [review]
Looks good enough to me :)

**Pull request merged**